kits/arm/ex_AR_kit.py

```diff
@@ -30,7 +30,7 @@
 
 # Mobile device setup
 phone_family = 'HEBI'
-phone_name = "Cal's iPhone"
+phone_name = "Mobile IO"
```

Comment:
I think the naming convention is `mobileIO` now. Could be wrong though... I always forget myself.

Reply:
Yes, it's `mobileIO`